src/AppBundle/Repository/EventWikiRepository.php

```diff
@@ -447,16 +447,16 @@ public function getSinglePageCreatedData(
         }
 
         $end = $end->format('YmdHis');
-        $usernamesSql = empty($usernames) ? '' : 'AND rev_user_text IN (:usernames)';
+        $usernamesSql = empty($actors) ? '' : 'AND rev_actor IN (:actors)';
 
         // Only use revision_userindex when filtering by user.
-        $userRevisionTable = empty($usernames) ? 'revision' : 'revision_userindex';
+        $userRevisionTable = $actors ? 'revision' : 'revision_userindex';
 
         $sql = "SELECT `metric`, `value` FROM (
                     (
-                        SELECT 'creator' AS `metric`, rev_user_text AS `value`
-                        FROM $dbName.revision
-                        WHERE rev_page = :pageId
+                        SELECT 'creator' AS `metric`, actor_name AS `value`
+                        FROM $dbName.revision, $dbName.actor
```

Comment:
I had to research this syntax, and apparently it's an older way to do JOINs. From what I'm reading ANSI-92 (newer) syntax is preferred. https://stackoverflow.com/a/1599201 What's your take? I realize this is an especially simple query

Reply:
Mhm, I'm kinda used to implicit joins - LMK if you'd like me to make it explicit and I'll commit a followup.